examples/Toy_segmentation/dltk_test.ipynb

```diff
@@ -0,0 +1,175 @@
iDwKBAMrlshRlNK+bTCbkcjkpmKrVqrw2uV2T\nyQTD4VAuzjabTbpZ/Drs7pFWzw7XysoKPvKRj+Atb3nLy55ELpdL3rO5XrsqlYrcXDDWxmKxwGKx\nSLHPm41SqQSn04lIJCJoDk3TEI/H0ev1UK/XhR1nNBoRDAaFNWU0GhGPx9HpdBAIBLC+vi5+QmJK\nWDBns1n5/IXDYdRqNcRiMaRSKdlCVVVVwLKMojp06BCm0ykWFhYwmUxw7NgxXH755TN5X0/l77C5\nTl/t27cPf/M3fzMvrOaaiV62uNJ1/Tde5K/e9yLP/18A/tcr+eIcMdArVa1WkUwmkc/nsbi4KNl7\nw+EQqqqi0+nI2IGhy/1+X7pEvKv3+/0olUoYjUbSLWK0CEdswWBQAIzsJAFALpcTdEOhUIDNZhPU\ngcViwWg0kotPo9GAyWSS4OV+v49KpSK8K3qouNru9/slSof4CL4+Keg0HjudTtl49Pv94rnxeDwS\neA3gBHgoR59kDbFbwZBmhj7TA8YLN4uw6XSKPXv24O1vfzt27dr1igqruU6u2C1i4VKv1xGPxwV7\n4Ha7EYvFpKAnssNsNstonJ3Y6XQKl8uFUCiEWCyGJ554Qopn5gWazWbZPHU6ndIpm06nCAaDqNfr\nKJfLiMVigmvgjQoJ8Pxc8CaFZnpiRlKplKQNzEKn8nfYXKefbrrpJgSDQXz2s59FPB6f9eHM9QbV\nTF3CRAUQesm18mQyKYT1druNYrGIlZUVVKtVqKoqxPTJZIIjR47g0ksvPSFY2Gg0wuVywefziW+E\nvi4WFhxBtttteL1eGZPQk+RyuTCZTKSTtbCwIAUNtxw5PqNRnn6pSqUiAE9GzgQCAdn6UxRFVtRJ\nomdoMrlStVpNOhiLi4sS1utwOJDNZjEajWRkYzQahU/UaDTg8/nQbrcxGo2kU8UNxuFwiE6ng2Aw\nCADitXG73RgMBkgkEvjEJz7xqn+W89Hg65eiKOj3++Ln4xIEFzDK5TLq9TrW1tYkZ3E4HMo4mOdD\no9EQLxQA4cIxAH1xcRFPPvkkdF1HoVAQTALH1DabTRY4mLvJiCN2r1RVlVB0bsgSzbG5uQmv1yv5\nhwCQTCZPt23Buc5SfeYzn8Ell1wy68OY6w2umQc3u1wuGAwG4eb0+30Bg9LrQYaTrutwuVwwmUzI\n5/Ow2+1wuVxoNpvI5XLQNA2TyQSdTkfWymu1GvL5vGzS8d8zvy0ajcLlcsnmHkGdk8kEDocD55xz\nDmKxGCaTCbrdLnw+HxwOh+SwMXqmXq/LyM7tdiOdTiOdTkuXgJ4ywktbrZZswJnNZgQCASGgt1ot\n9Pt9gYJyRMktQhZELpdLKPStVku6HQCEp0X+1mAwEKI237fJZIJcLicF5t69e3H++ee/pp+ly+Wa\nb9y8TnFzlAsLXDrg6Je+ukajgUwmIx0jj8eDbreLzc1NGI1GiUyi746vZTKZUKvV0O12ZfPWYDCg\nVqtJ6HMoFILD4cBkMkGhUJCNVE3TJNeQ3VKa6VutlkTpOJ1OZLNZhMNhGI1GNBoNOJ1OiZGaa65T\nqX379smN41xzzVIz71wxZJgFDH+hc5OPeYPsDpnNZmQyGUEc0Mje7/ehqqogFTgeA7Z8Wgx4ZjfI\n5/NBVVXY7XYAWwBH3lmHQiEMh0NUq1VZW+e/5cjE6XSKL4o+Fprpo9GoFIeBQECgnMw2VFUVR44c\ngc1mQzAYxOrqqozwTCYTqtWqjFSq1Sqq1ar4ohjiPBwOUalUAEC6HL1eDz6fT4z328nZ3HLkaj2B\nlQDQ7Xbhdruxb9++1zUKPJXdKxYEZ+vG4MGDB2XBgyPh7Zue7FCRLcURb7lclsxKFt/NZhPT6RR+\nv18yK+kPXFhYQKPREK8WA7w5UrTZbDh06JAY6P1+v2z6sejnec8bjc3NTWSzWRlNe71etFot6cgC\nWx227X7FueY62dq3bx+++MUvIplMvvyT5zqpeuCBB/DEE0/M+jBOK820c5XJZDAYDBCPx2G324WH\nQ3p4oVCQC8xgMEC5XJaxGi9Ew+FQttwY7UE/FDtTwWAQVqsVk8kElUpFNv04arPb7YjFYgiFQgAg\nY8PxeIxCoSBFDIGNHFcyY43bjuwAdDoduajxbp8XlsFgIPEzHIkOBgNsbm7Kc0wmk3S4thPiGQxN\nfxgAKTgJaex2u5hMJpKByKgfBkgrioJ6vS4bjVarFXv37sU111zzuj1WjGQ5FeLI9GzV3XffjWPH\njgkGg0VKNBqVtAAuR3BxgwWRyWRCOBxGIpEQPEkgEJBilx2x6XQq3VsWwRzl+f1+jEYj1Go1Cfsm\n32o8HqPZbEr3jD+LcrmMcrmMzc1NWY7I5/NwOBwoFAqSKhCJRDAcDrG0tDTrt3mus1T79u3DF77w\nBSwuLs76UN6Q+u53v4sf/ehHsz6M00ozLa5cLheGwyGazSZUVUWhUBBu1GAwkO5Sr9dDpVKBz+cT\nkjuLjVgsBlVV4Xa7USgUpOsDQOI4SDcnMJRbTd1uF81mE61WSwCMZPXQf+X1eiUvkMZ3jtz4OEds\nzOdjN85sNqPdbqPRaMhFT9d1dDodyTBsNptIp9NyweJGocPhkIgZxp20220Jn+ZYsd1uC/6h1+sh\nm82KaZ/r/Hy/Op2ObEvyOFwuF970pjfhfe97QW/vqxbZYHO9OrFTVa/XpWMLbHUD6fUj8NVkMkHT\nNKTTaZjNZmGS9Xo9bGxsyIiWm4I0my8uLkqBRZq/w+GAz+eD1+uF2+1Gv99HKBRCNBqVMR7jbAjV\n5c3NeDwWWKmqqlhaWpLPq67raLVaWF9fh81mk23CueY6FfqTP/mTefE+12mlmf62Y6gyAOmwEPTJ\nURxHE+PxGO12G7lcDpPJBNVqVUKGiXFg0dJut9HtdsWLxK2qVquFVquFTCYDq9WKUqkkYxH+2el
```

Comment:
Our dice currently only handles multi class cases - I managed to get the example working by changing everything to 2 classes. However, then the training data would be somewhat incorrect.